tests/e2e/specs/in-game/game-over/winConditions.spec.js

```diff
@@ -73,6 +75,7 @@ describe('Winning the game', () => {
   });
 
   it('Shows when player wins game with 0 points and four kings', () => {
+    cy.skipOnGameStateApi();
```

Comment:
Needing to do this for every test feels weird, impossible to use a before each and allowlist? Is there any sort of factor they all share we could better hook in to?

Reply:
I came up with a way to skip test dynamically based on if the endpoint existed or not in #1005, the problem I ran into was that some of the test hit endpoints that DID exist, but needed changes to work with the gamestate api

Reply:
putting this skipper in each test gives us alot of control.. it's pretty redundant, but some test hit multiple endpoints so some within a spec will work when specific endpoints get built, but not all the test will

Reply:
I agree with @Haviles04 . This way is the most explicit and the most flexible. This way we can disable individual tests and re-enable them as they are put into scope